base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi

```diff
@@ -463,7 +463,7 @@
       <macros>
         <include_parent_macros>true</include_parent_macros>
       </macros>
-      <visible>false</visible>
+      <visible>true</visible>
```

Comment:
Did you really mean to edit the Eurotherm OPI?

Reply:
Removed from branch

Reply:
- Fixed title casing

- Moved units selector to 'Device' tab instead of channel tabs